uiautomator2/__init__.py

```diff
@@ -873,6 +873,42 @@ def app_info(self, package_name: str) -> Dict[str, Any]:
             "versionCode": info.version_code,
         }
 
+    def app_auto_grant_permissions(self, package_name: str) -> bool:
+        """ auto grant runtime permissions to target app，prevent dynamic permission pop-up window to pop up
```

Comment:
ref <https://github.com/appium/appium-adb/blob/master/lib/helpers.js#L585>

```
if (apiLevel < 23) {
      log.debug(`Skipping permissions grant option, since ` +
                `the current API level ${apiLevel} does not support applications ` +
                `permissions customization`);
```

does here need to check?

Reply:
尝试在增加授权之前获取了目标App的target API版本，做判断之后再进行授权操作。